src/site/stages/build/process-cms-exports/transformers/node-va_form.js

```diff
@@ -5,7 +5,15 @@ const {
   utcToEpochTime,
 } = require('./helpers');
 
+const moment = require('moment');
+
+function toUtc(dateString) {
```

Comment:
Looks like we already have a similar function in `node-event.js`. Could that be shared? Any other places where we do something like this? (I don't have much knowledge of this code but wanted to mention it)

Reply:
Nevermind, looks like that code is removed in https://github.com/department-of-veterans-affairs/vets-website/pull/15953

Reply:
If we see that pattern show up again, we can always add it to `src/site/stages/build/process-cms-exports/transformers/helpers.js`.